extensions/kind/scripts/build.js

```diff
@@ -34,3 +36,6 @@ if (fs.existsSync(builtinDirectory)) {
 }
 
 zipper.sync.zip(path.resolve(__dirname, '../')).compress().save(destFile);
+mkdirp(unzippedDirectory).then(() => {
```

Comment:
this part will enable kind extension by default, so I'm unsure we want to have it enabled by default yet
cc @slemeur

Reply:
Reverted the commit